EducationalProgram.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+public class EducationalProgram
+{
+    static void Main()
+    {
+        StudentAndTeacherEntity studentAndTeacher1 = new StudentAndTeacherEntity("John", "Doe", new DateTime(1995, 5, 15), "123 Main St", 12345, "Computer Science", 3.75, 2, "Science");
```

Comment:
Make the first 4 rows in Main on start from one column

Reply:
not a problem, thanks for feedback!